WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Success card/Table View/MigrationSuccessCell+WordPress.swift

```diff
@@ -1,10 +0,0 @@
-import Foundation
-
-extension MigrationSuccessCell {
-
-    @objc(configureWithViewController:)
-    func configure(with viewController: UIViewController) {
-        // Adding an empty implementation of this method so Xcode doesn't complain.
-        // The whole `MigrationSuccessCell` shouldn't exist in WordPress. It's only needed for Jetpack.
-    }
-}
```

Comment:
Now this is redundant!

Yeah for less of this kind of confusing duplication!

Reply:
Yeah, it was a poor way to implement it.